schemas/configuration/v4/service_account_environment.json.in

```diff
@@ -38,6 +38,9 @@
         "irods_ssl_certificate_key_file": {"type": "string"},
         "irods_ssl_dh_params_file": {"type": "string"},
         "irods_ssl_verify_server": {"enum": ["cert","hostname","none"]},
+        "irods_tcp_keepalive_intvl_in_seconds": {"type": "integer"},
+        "irods_tcp_keepalive_probes": {"type": "integer"},
+        "irods_tcp_keepalive_time_in_seconds": {"type": "integer"},
```

Comment:
Just noting the docs will need to be updated to include descriptions for these too.

Reply:
Right, forgot to hit submit on that PR... https://github.com/irods/irods_docs/pull/209